R/pkg/R/DataFrame.R

```diff
@@ -3058,7 +3057,7 @@ setMethod("str",
 #' @note drop since 2.0.0
 setMethod("drop",
           signature(x = "SparkDataFrame"),
-          function(x, col, ...) {
```

Comment:
just to clarify removing `...` is intentional ? Just wondering as we have the `@param` documentation above